### PR TITLE
resolves PDB-309

### PR DESCRIPTION
**Pull request by steveblackmon**
used this branch to publish:
http://streams.peoplepattern.com.s3-website-us-east-1.amazonaws.com/streams/0.3.4-PP-SNAPSHOT/

### CODE REVIEW

File: components/streams-processor-peoplepattern/src/main/jsonschema/Miscellaneous.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
```
Comment (jfrazee) [inline components/streams-processor-peoplepattern/src/main/jsonschema/Miscellaneous.json]: Minor comment: these are surfaced as "extended demographics" in the app. Not sure if we want to continue with that terminology or not.

File: pom.xml [hunk 2]
```diff
@@ -57,15 +59,15 @@
...
     </distributionManagement>
 
     <repositories>
         <repository>
-            <id>s3.release</id>
-            <name>PP S3 Release Repository</name>
-            <url>s3://releases.peoplepattern.com</url>
+            <id>s3.release.http</id>
+            <name>PP S3 Release HTTPS</name>
+            <url>http://releases.peoplepattern.com.s3-website-us-east-1.amazonaws.com</url>
```
Comment (jfrazee) [inline pom.xml]: Just an idea but might be worth putting these in DNS -- see [OPS-193](https://peoplepattern.atlassian.net/browse/OPS-193).

**Comment by jfrazee** [comment]
:+1:

Your call on the miscellaneous vs. extended demographics thing.

File: components/streams-processor-peoplepattern/src/main/jsonschema/Miscellaneous.json (hunk 1 above)
Comment (steveblackmon) [inline components/streams-processor-peoplepattern/src/main/jsonschema/Miscellaneous.json]: cool, really just wanted some place in this repo to define these even though i dont think we have an API exposed for them

**Pull request merged**